tests/ct_pipeline.c

```diff
@@ -1,56 +1,51 @@
 #include "adapters/libevent.h"
 #include "hircluster.h"
+#include "test_utils.h"
 
 #include <assert.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
-#define CLUSTER_NODE "127.0.0.1:30001"
+#define CLUSTER_NODE "127.0.0.1:7000"
```

Comment:
Why change the ports? Just curious.

The references to these ports in the README needs to be updated.

Reply:
Updated readme.
A lot of examples and the redis docs uses 7000 as a base-port, like https://redis.io/topics/cluster-tutorial
Found that out when looking at https://github.com/Grokzen/docker-redis-cluster, which also uses 7000 as default